clang/lib/CIR/Dialect/Transforms/FlattenCFG.cpp

```diff
@@ -937,4 +1015,10 @@ std::unique_ptr<Pass> createFlattenCFGPass() {
   return std::make_unique<FlattenCFGPass>();
 }
 
+std::unique_ptr<Pass> createFlattenCFGPass(bool throughMLIR) {
```

Comment:
Could this function just be the previous one extended with a defaulted parameter, such as
```suggestion
std::unique_ptr<Pass> createFlattenCFGPass(bool throughMLIR = false) {
```
or `true`?